nerfstudio/scripts/render.py

```diff
@@ -149,14 +153,27 @@ def _render_trajectory_video(
                         )
                         sys.exit(1)
                     output_image = outputs[rendered_output_name]
-                    output_image = (
-                        colormaps.apply_colormap(
-                            image=output_image,
-                            colormap_options=colormap_options,
+                    if "depth" in rendered_output_name:
```

Comment:
I'm not a huge fan of adding special logic to outputs like this. It could be confusing to users why the "depth" output looks different than a custom output like "depth_median". Maybe we can add logic in the colormap options where a boolean could exist for `is_depth_colormap`.

Reply:
In this code `rendered_output_name` is a string, so maybe writing `if rendered_output_name.find("depth") != -1:` would be less confusing. Do you think that change would suffice? It follows similar logic we have throughout the codebase, although of course it's not ideal. By default, the colormaps should look the same since the depth min and max are set to None.

Reply:
I'm going to do a check with `is_depth = rendered_output_name.find("depth") != -1`. I think modifying ColormapOptions isn't so easy now that it's passed throughout `render.py`.